instrumentation-api-incubator/src/main/java/io/opentelemetry/instrumentation/api/incubator/semconv/rpc/RpcAttributesGetter.java

```diff
@@ -24,4 +24,20 @@ public interface RpcAttributesGetter<REQUEST> {
 
   @Nullable
   String getMethod(REQUEST request);
+
+  default int getClientRequestSize(REQUEST request) {
+    return 0;
+  }
+
+  default int getClientResponseSize(REQUEST request) {
+    return 0;
+  }
+
+  default int getServerRequestSize(REQUEST request) {
+    return 0;
+  }
+
+  default int getServerResponseSize(REQUEST request) {
+    return 0;
+  }
```

Comment:
the type for these should be `Long`, and these should return `null` by default. We need to consider whether this should be split into separate client/server getters similarly to what we do for http client/server. Would it make sense to introduce a response parameter too? Or perhaps just have `getRequestSize` and `getResponseSize` in the getter, we already have separate extractors for client/server these could be used to set the correct attribute.

Reply:
Thank for your suggestion, Done.